### DISCUSSION TITLE
Gemstone for casual players

### DISCUSSION THREAD

**Discussion by player9689** [DiscordDungeons/Bugs#977] [Ideas]
I have a idea for people that dont like dungeon.
So to still offer those players the use of lux and enchant i think i have the solution.

Enchant chest u get from adv with 1 lux and u turn them into dungeon chest.
So u have same chances as in dungeon for gemstone.
I roughly think paying 1 lux for 1% at any gemstone would be decent balance.

**Comment by Feathers8** [comment]
Don't you get like 8-12 lux per day?

**Comment by Sven65** [comment]
Gems provide a large boost to items, and the effort required to get them should reflect that.

Using lux on chests (which have about a 1% drop rate by the way), wouldn't properly reflect the effort we require to get a gemstone.
